pipeline/cloud/schemas/__init__.py

```diff
@@ -22,18 +22,32 @@ class BaseConfig:
 
 
 class BaseModel(_BaseModel):
+    """Base model for schemas."""
+
     class Config(BaseConfig):
-        orm_mode = True
-        use_enum_values = True
+        pass
 
-
-class Patchable(BaseModel):
-    class Config:
-        extra = Extra.forbid
+    @validator("*", pre=True)
+    def convert_to_utc(cls, value):
+        if isinstance(value, datetime):
+            return value.astimezone(timezone.utc)
+        return value
```

Comment:
nice, this looks a neat way to do it. I presume this will just ensure all 'naïve' datetimes will have a UTC timezone attached to them?

Reply:
Yeah exactly. It is a little implicit though.. but better than enforcing that timezones be set pre-serialisation/deserialisation